goal_a: -4.1;1.15;0.0;1.0
upper threshold: 0.99
lower threshold: 0.3
charging_dock: 0.386;4.669;0.0;1.0
goal_b: -2.504;4.455;0.0;1.0